src/averageAges.js

```diff
@@ -52,7 +56,14 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  let child = people.filter(baby => !onlyWithSon
```

Comment:
![image](https://user-images.githubusercontent.com/35385361/102813376-5f3d2280-43d1-11eb-9982-ca4eab00437e.png)
```suggestion
  let children = people.filter(baby => !onlyWithSon
```